Telegraf docker collector metrics initial commit with integration (#1)
logzio/telegraf-docker-collector-metrics: README.md

# Telegraf Docker Metrics Collector

> To simplify shipping metrics from one or many sources, we created Docker Metrics Collector. Docker Metrics Collector is a container that runs Telegraf collector.

Reviewer: ```suggestion
To simplify shipping metrics from one or many sources, we created Telegraf Docker Metrics Collector. Telegraf Docker Metrics Collector is a container that runs Telegraf collector.
```

> ## Pull the Docker image of the Telegraf Docker Metrics Collector

Reviewer: ```suggestion
## 1. Pull the Docker image
```

```sh

> docker pull logzio/telegraf-docker-collector-metrics

Reviewer: ```suggestion
docker pull logzio/telegraf-docker-collector-metrics:latest
```

```

> ## Run the Telegraf Docker Metrics Collector

Reviewer: ```suggestion
## 2. Run the collector
```

```sh
docker run --name telegraf-docker-collector-metrics \
 --env METRICS_TOKEN="<<METRICS-SHIPPING-TOKEN>>" \
 --env LOGZIO_LISTENER="<<LOGZIO_LISTENER>>" \
 -v /var/run/docker.sock:/var/run/docker.sock \
 logzio/telegraf-docker-collector-metrics:latest
```

If you prefer to store these environment variables in an [`.env` file](./docker.env), run the following command:

```sh
docker run -d --env-file=docker.env -v /var/run/docker.sock:/var/run/docker.sock logzio/telegraf-docker-collector-metrics:latest
```

| Name            | Description                                                                                                               |
| --------------- | :------------------------------------------------------------------------------------------------------------------------ |

> | METRICS_TOKEN   | Your Logz.io metrics account token. Replace <<METRICS-SHIPPING-TOKEN>> with the token of the account you want to ship to. |

Reviewer: ```suggestion
| METRICS_TOKEN   | **Required**. Your Logz.io metrics account token. Replace <<METRICS-SHIPPING-TOKEN>> with the token of the account you want to ship to. |
```

> | LOGZIO_LISTENER | Your Logz.io listener address, For example: `https://listener.logz.io:8053`.                                              |

Reviewer: ```suggestion
| LOGZIO_LISTENER | **Default**: `https://listener.logz.io:8053`. Your Logz.io listener address followed by port `8053`.                                             |
```

> | DOCKER_ENDPOINT | Address to reach the required Docker (default: `unix:///var/run/docker.sock`)Daemon.                                      |

Reviewer: ```suggestion
| DOCKER_ENDPOINT | **Default**: default: `unix:///var/run/docker.sock`. Address to reach the required Docker Daemon.                                      |
```

> | TIMEOUT         | The request timeout for any Docker Daemon query ( default: `5s`).                                                         |

Reviewer: ```suggestion
| TIMEOUT         | **Default**: `5s`. The request timeout for any Docker Daemon query.                                                         |
```

> 

Reviewer: This newline breakes the table

Author: **Can you explain what is broke?**
@mirii1994 

![screencapture-github-logzio-telegraf-docker-collector-metrics-blob-intergation-stage-README-md-2022-05-18-23_41_04](https://user-images.githubusercontent.com/11467909/169152201-dac6ca18-b227-4956-bc11-c27c59716200.png)

Reviewer: My bad. You're right the table looks good.

> | EXCLUED_IMAGES | A list of strings, [regexes](https://pkg.go.dev/regexp), or [globs](https://github.com/gobwas/glob) whose referent container image names will not be among the queried containers. !-prefixed negations are possible for all item types to signify that only unmatched container image names should be monitored. For example:`imageNameToExclude1,imageNameToExclude2` ( default: `nil`) |

Reviewer: ```suggestion
| EXCLUED_IMAGES | **Default**: `nil`. A list of strings, [regexes](https://pkg.go.dev/regexp), or [globs](https://github.com/gobwas/glob) whose referent container image names will not be among the queried containers. !-prefixed negations are possible for all item types to signify that only unmatched container image names should be monitored. For example:`imageNameToExclude1,imageNameToExclude2`) |
```

> ### Check Logz.io for your metrics

Reviewer: ```suggestion
### 4. Check Logz.io for your metrics
```
